src/static/js/overlay.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { documentReady, startBpmnVisualization, addOverlays } from '../../index.es.js';
+
+// to show mouse pointer position - usefull for testing
```

Comment:
❓ can we move this function in a a dedicated file to avoid duplication with the rendering-diagram.js (?) file?

Reply:
Yes, I though the same thing, but I didn't want to have to many refactoring. But we can do it :)

Reply:
We do that later when we'll add navigation with overlay to verify that the overlay is always display well.